### PR TITLE
Resolves #620

### LINKED ISSUE #620: New script idea:  Interview Information Sheet
A script to produce an information sheet of high-level case and general information that can be used as a resource when meeting with a client in person.  The top half of the sheet is for the worker to use as a reference and to document any updates that need to be made to the case.  The bottom half of the sheet can be given to the client as a resource.  

For greater efficiency, this script makes use of classes.  We may want to consider whether it makes sense to add the classes to the Functions Library.

A functional draft is ready to pull request.  I am attaching the Word template the script uses.
[Interview Information Sheet.docx](https://github.com/MN-Script-Team/DHS-PRISM-Scripts/files/572461/Interview.Information.Sheet.docx)

### PR DESCRIPTION
**Pull request by levesswe**
Resolves #620 - BLIP: Script produces an information sheet of case and resource information for use at client in-person visits.

### CODE REVIEW

File: actions/interview-information-sheet.vbs [hunk 1]
```diff
@@ -0,0 +1 @@
...i    .FormFields("case_number").Result = PRISM_case_number   .FormFields("other_cases").Result = info.number_of_cps_open_cases    .FormFields("function").Result = info.case_function   .FormFields("assigned_worker").Result = info.worker_name & ", Phone: " & info.worker_phone   .FormFields("last_payment").Result = info.last_payment_date & " " & FormatCurrency(info.last_payment_allocation) & " " & info.last_payment_type    .FormFields("arrears_balance").Result = FormatCurrency (info.pa_arrears) & " PA arrears + " & FormatCurrency(info.npa_arrears) & " NPA arrears = " & FormatCurrency(info.ttl_arrears) & " total"   .FormFields("pay_plan").Result = info.pay_plan_info(PRISM_case_number)   .FormFields("cch").Result = FormatCurrency (Ccur(info.cch_amount))   .FormFields("cms").Result = FormatCurrency(CCur(info.cms_amount))   .FormFields("cmi").Result = FormatCurrency(CCur(info.cmi_amount))   .FormFields("ccc").Result = FormatCurrency(CCur(info.ccc_amount))   .FormFields("csp").Result = FormatCurrency(CCur(info.csp_amount))   .FormFields("arrears_payback").Result = FormatCurrency(info.monthly_non_accrual)   .FormFields("ncp_mci").Result = info.ncp_mci  End WithEND IFscript_end_procedure("")
```
Comment (theVKC) [inline actions/interview-information-sheet.vbs]: Somehow, this came in as a single line without line breaks. @levesswe: can you check your setup?

**Comment by levesswe** [comment]
When I do a pull request, under wrapping, it only gives me two options, no wrap or soft wrap, and soft wrap is just as hard to read.

**Comment by theVKC** [comment]
It shouldn't be about the wrap, it's about your editor. Where are you pasting this from? For some reason, your copy/paste appears to be blocking the newline character. Word wrap is just a visualizer, we're talking about line breaks.

**Comment by levesswe** [comment]
I am pasting it from the BlueZone Script Host, and it looks fine in the BlueZone Script Host.

Wendy LeVesseur
Child Support Officer

Anoka County Government Center
2100 3rd Avenue, Suite 401
Anoka, MN 55303
763-422-7323

**Comment by theVKC** [comment]
I'm wondering if something goofy is going on there. Can you try and paste it into notepad, and see if you get the same result?

**Comment by levesswe** [comment]
I am getting the same result.

It seems to paste okay on the screen, but when saved, it is saved without the spacing.

Wendy LeVesseur
Child Support Officer

Anoka County Government Center
2100 3rd Avenue, Suite 401
Anoka, MN 55303
763-422-7323

**Comment by theVKC** [comment]
Hmmmmmm. I am wondering if this is some goofy setting on your computer. I've never seen anything like this before.

@heatherallen is soon to pull a pull request, perhaps if she's experiencing something that might help us figure it out.

My suggestion would be to try using an alternate editor (Atom is preferred, but Notepad++ is good too). Can you see if either of these editors produce the same error?

**Comment by levesswe** [comment]
I tried Notepad + + with the same results.  I suppose we will learn more after someone else does a pull request.

**Comment by levesswe** [comment]
So, FYI, I think the trouble is with using Internet Explorer.  I was successful when I used Google Chrome to do the pull request.

**Comment by theVKC** [comment]
Wow! Good to know! What version of Internet Explorer are you-all using in Anoka? That'd help in my documentation work, I might add it to the guidelines (#597).

**Comment by levesswe** [comment]
Internet Explorer 11

**Comment by RoberT-1000** [comment]
...Internet Explorer...Thanks a lot, Obama...

**Comment by theVKC** [comment]
@levesswe: we changed the script format and whatnot. Any chance you can update this one to match the other scripts in the repo and resubmit?

**Pull request closed without merging**